### PR TITLE
Set up Hello Microverse Project

### PR DESCRIPTION
**Pull request by mbonabucya**
### Hello-Microverse 

- [ ] Add index.html
- [ ] Add style.css file
- [ ] add styling header in the index.html
- [ ] Add .gitinore file
- [ ] Create test.md file
- [ ] create .github directory
- [ ]  Add workflows folder into .github
- [ ] Create linters.yml file

### CODE REVIEW

**Comment by mbonabucya** [comment]
Added Linters Configuration 
Installed latest version of Node
Added View port values in the Index.html

File: Html/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="css/style.css">
```
Comment (Chimwemwe-127001) [inline Html/index.html:7]: - [ ] I suggest you correctly link your HTML to your CSS. Try this ```<link rel="stylesheet" href="../css/style.css">```

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
...
\ No newline at end of file
+# Hello-Microverse
+
+
+-Index.html located in Html folder</br>
+-style.css is located in the css folder</br>
+-Added .gitignore file that will list the files to be ignored</br>
+-test.md will ignored</br>
+-Added Linters for syntax error check.
+
```
Comment (Chimwemwe-127001) [inline README.md]: - [ ] For your `README.md`, consider using the Microverse README template. It is of great importance that all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) be present in your repo.

**Review by Chimwemwe-127001 — CHANGES_REQUESTED** [review]
Required Changes ♻️ :warning: 
### Hi @mbonabucyahi
Good job so far! :tada: 

There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand: 

so close

## Highlight :+1: 
- Linters configured perfectly :+1:

## Required Changes ♻️
- [ ] I strongly suggest you add a summary and descriptive title for your pull request. Its important for pull requests to have a summary and descriptive title as it helps to describe introduced changes in more detail. [NOTE] Configuring linters isn't the only thing you are doing.
- [ ] Consider taking a look at your branch name. Branch should have a name of a feature or action. In our case ('`hello_microverse`' or '`hello-microverse`').

***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request closed without merging**